README.md

```diff
@@ -70,6 +70,8 @@ The `yarn build` step will download the latest TinyMCE package and generate new
 
 The TinyMCE API documentation is maintained within the [TinyMCE project repository](https://github.com/tinymce/tinymce) and compiled for the documentation site using [MoxieDoc](https://github.com/tinymce/moxiedoc).
 
+Running `yarn build` will download the latest TinyMCE package and generate new API reference content from source code. This API documentation must be generated and updated on the documentation site with each release.
```

Comment:
```suggestion
To update the published API documentation, change the version in the `.api-version` file, run `yarn build` and then commit the changes. Running `yarn build` will download the TinyMCE package specified in `.api-version` and generate new API reference content from source code.

**Note:** The API documentation should never be edited manually.
```
We don't need the last sentence and it doesn't download the latest version so that part isn't quite correct. Also we're missing the details about committing the changes to make it appear, so I think the above should address those issues.

Reply:
Thanks @lnewson